dev-python/pyasn1/pyasn1-0.6.0.ebuild

```diff
@@ -11,7 +11,7 @@ inherit distutils-r1 pypi
 DESCRIPTION="ASN.1 library for Python"
 HOMEPAGE="
 	https://pypi.org/project/pyasn1/
-	https://github.com/etingof/pyasn1/
+	https://github.com/pyasn1/pyasn1/
```

Comment:
I think it would make more sense to update it after bumping. After all, the old version comes from the old homepage.

Reply:
The last version from the old homepage is 0.4.8, everything after has been from pyasn1/pyasn1.